dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2492,7 +2492,7 @@ def test_graph_size_pyarrow(tmpdir, engine):
     ddf1.to_parquet(fn, engine=engine)
     ddf2 = dd.read_parquet(fn, engine=engine)
 
-    assert len(pickle.dumps(ddf2.__dask_graph__())) < 25000
+    assert len(pickle.dumps(ddf2.__dask_graph__())) < 7000
```

Comment:
This thing is what alerted me of the graph size increase over in https://github.com/dask/dask-expr/pull/1159

With https://github.com/dask/dask-expr/pull/1159 as is, this will go up to ~85KB from ~5KB right now, i.e. an increase of about 17x